src/views/instructor/training/EditSession.vue

```diff
@@ -4,7 +4,7 @@
 			<span class="card-title">
 				Enter Session Notes
 			</span>
-			<div class="loading_container" v-if="!session">
+			<div class="loading_container" v-if="!session === null">
```

Comment:
What is this line here? Are we trying to check if session is null?

Reply:
Nevermind I see. I think this is better written as session !== null but I'm not gonna hold up the review.